<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ColorWidget</class>
 <widget class="QWidget" name="ColorWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>381</width>
    <height>122</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ColorWidget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Background Color</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ColorLabel" name="backLabel">
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="2">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Text Color</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ColorLabel" name="textLabel">
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="3">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Grid Color</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ColorLabel" name="gridLabel">
       <property name="text">
        <string>gridLabel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" rowspan="2" colspan="4">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Select Colors</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" rowspan="2">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>180</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="2">
    <widget class="QPushButton" name="apply">
     <property name="text">
      <string>Apply </string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ColorLabel</class>
   <extends>QLabel</extends>
   <header>colorlabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
